src/current/v24.3/cockroachdb-feature-availability.md

```diff
@@ -47,6 +47,14 @@ Any feature made available in a phase prior to GA is provided without any warran
 **The following features are in preview** and are subject to change. To share feedback and/or issues, contact [Support](https://support.cockroachlabs.com/hc).
 {{site.data.alerts.end}}
 
+### Admission control for ingesting snapshots
+
+The [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) `kvadmission.store.snapshot_ingest_bandwidth_control.enabled`, which allows you to to optionally limit the disk impact of ingesting snapshots on a node, is in Preview.
+
+### Admission control to limit the bandwidth for a store
+
+The [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}), `kvadmission.store.provisioned_bandwidth`, which allows you to optionally limit the disk bandwidth capacity of stores on the cluster. Disk bandwidth admission control paces background disk writes to keep disk bandwidth within its provisioned bandwidth, is in Preview.
```

Comment:
first sentence is missing the "is in preview" clause e.g. "the foo, which allows you to bar, is in preview"

Reply:
oh i see the "is in Preview" is at the end of the second sentence.  i think it just needs to go on the end of the first sentence.

Reply:
Done